docs/extending-documentation.md

```diff
@@ -0,0 +1,289 @@
+
+> For the JavaScript code snippets, it only transforms the ones that use [MDJS](https://open-wc.org/mdjs/) syntax, e.g. \`\`\`js script, \`\`\`js story and \`\`\`js preview-story
+
+To ensure babel actually runs for your Storybook, you will have to add it to your es-dev-server configuration:
+
+`.storybook/main.js`:
+
+```js
+module.exports = {
+  stories: ['../node_modules/@lion/tabs/README.md', '../packages/**/!(*.override)*.md'],
+  esDevServer: {
+    nodeResolve: true,
+    watch: true,
+    open: true,
+    babel: true,
+  },
+};
+```
+
+You should now see the `LionTabs` instances transformed into your own `LeaTabs`!
+
+## Adjust documentation content
+
+In some cases you don't want to show all examples of how to use a component. Sometimes information is `Lion` specific, or perhaps in your design system you are not allowed to use a certain feature that we documented in `Lion`.
+
+In our example, we will show you have to remove the `Rationale` section that you would normally inherit from the `Lion` documentation.
+
+For this step we make use of a remark plugin for the MD content, similar to how you would use a babel plugin for JS content. It is called [Remark extend](https://lion-web-components.netlify.app/?path=/docs/tools-remark-extend--page).
+It will let you add, remove or replace sections or specific words.
+
+First of all we need to add the plugin to the `.storybook/main.js`:
+
+```js
+const fs = require('fs');
+const { remarkExtend } = require('remark-extend');
+
+function isLion(filePath) {
+  return filePath.indexOf('@lion/') !== -1;
+}
+
+function getLocalOverridePath(filePath, root = process.cwd()) {
+  const rel = filePath.substring(filePath.indexOf('/@lion/') + 7, filePath.length - 3);
+  return `${root}/packages/${rel}.override.md`;
+}
+
+module.exports = {
+  [...],
+  setupMdjsPlugins: (plugins, filePath) => {
+    if (!isLion(filePath)) {
+      return plugins;
+    }
+    const newPlugins = [...plugins];
+    const markdownIndex = newPlugins.findIndex(plugin => plugin.name === 'markdown');
+    const overridePaths = [`${process.cwd()}/.storybook/all.override.md`];
+    overridePaths.push(getLocalOverridePath(filePath));
+
+    let i = 0;
+    for (const overridePath of overridePaths.reverse()) {
+      if (fs.existsSync(overridePath)) {
+        const extendMd = fs.readFileSync(overridePath, 'utf8');
+        newPlugins.splice(markdownIndex, 0, {
+          name: `remarkExtend${i}`,
+          plugin: remarkExtend.bind({}),
+          options: { extendMd, filePath, overrideFilePath: overridePath },
+        });
+      }
+      i += 1;
+    }
+    return newPlugins;
+  },
+  [...],
+};
+```
+
+In the code above we have 2 places in where you can do overrides: `./.storybook/all.override.md` for generic overrides and via `getLocalOverridePath` for each component, when needed, the `rel` needs to be the same in `lion` and your own project to be able to override the right file.
+
```

Comment:
The code is still a little magical without more explanation.

Might wanna explain a bit more what setupMdjsPlugins means, and what we actually do here. 

Essentially, we figure out where the "markdown" Mdjs plugin is, and insert our own remark-extend plugins after it, one for each of the override paths that we define (in our case, only 2, local and global override). And the reason we do a .reverse() is because we reverse the order of override.md plugins, so that local overrides are done before the global ones (all.override.md).

So maybe it's nicer to do:
```js
const overridePaths = [`${process.cwd()}/.storybook/all.override.md`, getLocalOverridePath(filePath)];
```
and 
```js
for (const overridePath of overridePaths) { }
```
to make it a little bit more simpler code for our reader.